buildenv/jenkins/variables/defaults.yml

```diff
@@ -533,57 +543,18 @@ x86-64_linux_criu:
   extends: ['x86-64_linux', 'criu']
   node_labels:
     build:
-      11: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
-      17: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
-      18: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
-      next: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
+      all: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
     docker_image:
+      8: ''
       11: ''
   build_env:
     cmd:
+      all: 'source /home/jenkins/set_gcc_10.3.0_env'
+      8: 'source /home/jenkins/set_gcc7.5.0_env'
       11: 'source /home/jenkins/set_gcc7.5.0_env'
-      17: 'source /home/jenkins/set_gcc7.5.0_env'
-      18: 'source /home/jenkins/set_gcc7.5.0_env'
-      next: 'source /home/jenkins/set_gcc7.5.0_env'
 ppc64le_linux_criu:
   extends: ['ppc64le_linux', 'criu']
 s390x_linux_criu:
   extends: ['s390x_linux', 'criu']
 aarch64_linux_criu:
   extends: ['aarch64_linux', 'criu']
-#========================================#
-# Linux x86 64bits compiled with GCC 10
-#========================================#
-x86-64_linux_gcc10:
```

Comment:
These gcc10 specs also need to be removed from Pipeline-Build-Test-All.groovy
Are we ready to build with jdk11? If so, pls open another PR to add jdk11 gcc specs.

Reply:
Pipeline-Build-Test-All.groovy has been updated.

Reply:
You mean gcc11? 
We're ready for p, z & x linux, a linux depends on https://github.com/adoptium/infrastructure/issues/2538. New PR for to follow shortly.

Reply:
Yes I meant gcc11.